ddtrace/profile/__init__.py

```diff
@@ -0,0 +1,23 @@
+import sys
+
+import pkg_resources
+
+from ddtrace.profile import _build
+
+try:
+    __version__ = pkg_resources.get_distribution(__name__).version
+except pkg_resources.DistributionNotFound:
+    # package is not installed
+    __version__ = "unknown"
+
+
+def _not_compatible_abi():
+    raise ImportError("Python ABI is not compatible, you need to recompile this module")
+
+
+if _build.compiled_with <= (3, 7, 3):
+    if sys.version_info[:3] > (3, 7, 3):
```

Comment:
you probably don't need `[:3]` here.

```
>>> (2, 1, 0) >= (2, )
True
>>> (2, 1, 0, 'test') >= (2, )
True
>>> (2, 1, 0, 'test') >= (2, 1)
True
>>> (4, 'rc.1') >= (2, 1)
```

Reply:
Is there a specific reason to have it?

(btw, this isn't a blocker for me, just a curiosity)

Reply:
Yes, if you have this:
```
>>> (3, 7, 3, 'test') > (3, 7, 3)
True
```
Then the code will consider that your version is greater than 3.7.3, which is not the case for what I want here.

So:

```
>>> (3, 7, 3, 'test')[:3] > (3, 7, 3)
False
```

Problem solved.